Form-Controls/index.html

```diff
@@ -13,15 +13,46 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label>Name</label>
+          <input type="text" name="name" id="name" required minlength="2" title="Name must be at least 2 characters">
+
+
+        </div>
+        <br>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required>
+        </div>
+        <br>
+        <div>
+          <label for="colors">Choose a color:</label>
```

Comment:
If the user doesn't actively select a color, they will get 'Blue' by default. Can you think of a better implementation that will make sure that the user interacts with this dropdown?